README.md

```diff
@@ -1,268 +1 @@
-    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS3</a> custom properties, <a href="https://css-tricks.com/snippets/css/a-guide-to-flexbox/">FlexBox</a>, <a href="">Grid</a> to make the website visually attractive</li><br>
-    <li><a href="https://reactjs.org/">Javascript</a></li><br>
-    <li><a href="https://mozilla.github.io/addons-linter">Linters</a> for coding convention and coding formating</li>
-      <li><a href="https://webpack.js.org/guides/getting-started/#basic-setup">Webpack</a> </li>
-    
-  </ul>
-</details>
-
-<details>
-  <summary>Javascript runtime environment</summary>
-  <ul>
-    <li><a href="https://nodejs.org">Node JS</a> as a javascript runtime machine [git], a free and open source distributed version control system</li>
-  </ul>
-</details>
-
-<details>
-  <summary>Deployment</summary>
-  <ul>
-    <li><a href="#deployment">GitHub Pages</a></li>
-  </ul>
-</details>
-
-<br><hr>
-
-<!-- KEY FEATURES -->
-### <b>Key Features 🏷️</b> <a name="key-features"></a>
-
-- [x] **List Structure**
-- [x] **Add items**
-- [x] **Remove items**
-- [x] **Edit todo item**
-- [x] **Cross out completed todo items**
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-<br><hr>
-
-<!-- LIVE DEMO -->
-## <b>Live Demo 🚀</b><a name="live-demo"></a>
-
-This project is hosted on [GitHub pages](#deployment). The Live application can be found [here](https://rileymanda.github.io/todo-list/dist).
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-<hr>
-
-<!-- GET STARTED -->
-## <b>Getting Started 💻</b><a name="getting-started"></a>
-
-To get a local copy up and running, follow these steps.
-
-<!-- PREREQUISITIES -->
-### <b>Prerequisites 🧱</b><a name="prerequisites"></a>
-
-In order to run this project you need:
-
-- A Github account
-- Node JS
-- A web browser to view output e.g [Microsoft Edge](https://www.microsoft.com/en-us/edge).
-- An IDE e.g [Visual studio code](https://code.visualstudio.com/).
-- [A terminal](https://code.visualstudio.com/docs/terminal/basics).
-
-<br><hr>
-
-<!-- SETUP -->
-### <b>Setup ⚙️</b><a name="setup"></a>
-
-Clone this repository to your desired folder:<br>
-```sh
-  cd my-folder
-  git clone https://github.com/RileyManda/todo-list
-
-  ```
-  - Navigate to the folder
-```
-cd todo-list 
-```
-- Checkout the branch
-```
-git checkout main
-```
-<br><hr>
-
-<!-- INSTALL -->
-### <b>Install 🏗️</b><a name="install"></a>
-
-Install all dependencies:
-All the dependencies are available in package.json and can be installed locally by running:
-```sh
-    npm install
-```
-<br><hr>
-
-<!-- USAGE -->
-### <b>Usage 📂</b><a name="usage"></a>
-
-To run the project, in your terminal, execute the following commands:
-
-```sh
-    npm run build   npm start
-```
-<!-- TESTS -->
-<br><hr>
-### <b>Run tests 🧪</b><a name="run-tests"></a>
-
-How should it be tested? 
```

Comment:
Your project functions as expected, You just need to make some changes to your Readme file  please add testing command  for testing your project code . 👍💯